src/test/java/uk/gov/hmcts/cmc/claimstore/controllers/ClaimControllerTest.java

```diff
@@ -46,7 +46,9 @@ public void setup() {
     public void shouldSaveClaimInRepository() {
         //given
         ClaimData input = SampleClaimData.validDefaults();
-        when(claimService.saveClaim(eq(USER_ID), eq(input), eq(AUTHORISATION), eq(singletonList("admissions"))))
+        when(claimService
+            .saveClaim(
+                eq(USER_ID), eq(input), eq(AUTHORISATION), eq(singletonList("admissions"))))
```

Comment:
I saw a discussion about features not being needed, is there a reason "admissions" and so on is still present?

Reply:
I had a chat with @ckalista and @sabahirfan about this, for saveLegalClaim i removed features and we have to do some clean up later to remove features for saveClaim but for now we may need it because we were using DQ feature in our very recent release.